crates/sui-framework/sources/crypto/range_proof.move

```diff
@@ -0,0 +1,18 @@
+// Copyright (c) 2022, Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+module sui::range_proof {
```

Comment:
as we do algorithmic separation of the crypto modules in this PR, I'd call this `bulletproofs` instead of `range_proof` for consistency.